workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.test.ts

```diff
@@ -1535,6 +1536,33 @@ describe('DynamoDBService', () => {
       expect(generatedParams).toEqual(expectedParams);
     });
   });
+
+  describe('updateExecuteAndFormat', () => {
+    let unmarshalledAttributes: Record<string, JSONValue>;
```

Comment:
Any reason why it's not `Record<string, string>` ?

Reply:
Because the value of the attribute could be a number, a list, etc. Tyler wrote this JSONValue type to capture the possibilities.